scripts/qesap/lib/process_manager.py

```diff
@@ -31,6 +32,11 @@ def subprocess_run(cmd, env=None):
         return (proc.returncode, [])
     stdout = [line.decode("utf-8") for line in proc.stdout.splitlines()]
 
+    print(f"OUTPUT START FOR COMMAND: {cmd}")
+    sys.stdout.flush()
     for line in stdout:
-        log.debug('OUTPUT: %s', line)
+        print(line)
+        sys.stdout.flush()
+    print(f"OUTPUT END FOR COMMAND: {cmd}")
+    sys.stdout.flush()
```

Comment:
This part needed to change, otherwise the logging messages were mixed with the json output. This was strange and shouldn't happen, since this whole execution is serial and not parallel, but I think that the `print` commands stacked up instead of being immediately delivered to stdout, so at some point some of the latter logging messages were mixed in. 

This was fixed by using sys.stdout.flush() after each line printing, to send it to stdout immediatelly, so... I was probably right? Dunno. But it works.

Reply:
nice master!!